### ISSUE TITLE
Tooltip default triggers

### ISSUE THREAD

**Issue by akhchan99** [bootstrap-vue/bootstrap-vue#4425]
Is there anyway to override the default triggers to just "hover" (instead of both focus and hover) for the whole application instead of overriding it on each reference to v-b-tooltip?

**Comment by tmorehouse** [comment]
Unfortunately no.  Mainly due to accessibility reasons, as tooltips cannot be triggered by keyboard navigation when the trigger is set to `hover` only.

If using the component versions, you can create functional wrapper components that change the default triggers (as well as other props):

```js
import BTooltip from 'bootstrap-vue'

export default {
  name: 'MyTooltip',
  extends: BTooltip,
  props: {
     triggers: {
       default: 'hover'
     }
  }
}
```

**Comment by tmorehouse** [comment]
Similarly, one could "Extend" (wrap) the directive version (albeit with a bit more code) to set the 'hover` modifer to true, and create a new custom directive `v-my-tooltip`

**Comment by akhchan99** [comment]
ic, thanks for the workarounds
